decisions.md

```diff
@@ -2,23 +2,43 @@
 layout: default
 ---
 
-## The Good Docs Project Decisions
+# Decisions
 
-### Criteria for Decision Making
+This page describes the decision making processes used within The Good Docs Project.
+
+**Version:** 2.0
+
+**Last updated:** August 2020
+
+## Voting
 
 Decisions are based on voting using the following:
 
-    +1 : For
-    +0: Mildly for, but mostly indifferent
-    0: Indifferent
-    -0: Mildly against, but mostly indifferent
-    -1 : Strongly against. This is expected to be accompanied by a reason.
+*   +1 : For
+*   +0: Mildly for, but mostly indifferent
+*   0: Indifferent
+*   -0: Mildly against, but mostly indifferent
+*   -1 : Strongly against. This is expected to be accompanied by a reason.
+
+## Simple decisions
+
+For most decisions, a motion is raised within one of The Good Docs Project open forums (slack, email list, weekly meeting, etc.) and those present vote and decide. 
+
+## Significant decisions
+
+Should a decision:
+
+*   Be of interest to others not present,
+*   Be strategic or impact many within the project,
+*   Be contentious, and after discussion at least one member is still strongly against the motion, by continuing to vote -1,
+
+then a motion should be raised as an email thread on our [general email list](https://thegooddocsproject.groups.io/g/main). The title should start: “Motion: …”
 
-For most decisions, a motion is raised within one of TheGoodDocsProject open forums (slack, email list, etc) and those present vote and decide. Should a decision:
+All community members are encouraged to vote to demonstrate community support, however contentious motions will only count votes from PSC members.
```

Comment:
This could be a disincentive to non-PSC members to get involved.

Reply:
Yes, it could dissuade people from voicing their opinion. We welcome respectful and robust debate. I'm not sure how to change the wording to resolve this though.